src/pages/api/search.js

```diff
@@ -12,9 +16,10 @@ export default async function findCoords(req, res) {
       },
     },
   );
-  if (result === undefined) {
-    res.send({});
-  } else {
-    res.send(result.geometry.location);
-  }
+  // if (result === undefined) {
+  //   res.send({});
+  // } else {
+  //   res.send(result.geometry.location);
+  // }
+  res.send(resultfromApi.data.results[0].geometry.location);
```

Comment:
No, don't do that. The purpose of doing it the way you commented out is to prevent runtime errors if there are no results. The way you're currently doing it, this will break in that situation because `results[0]` is undefined.